web/main/tasks.py

```diff
@@ -0,0 +1,66 @@
+from celery import shared_task
+from celery.signals import task_failure
+import socket
+from time import sleep
+
+from django.core.mail import mail_admins
+
+import logging
+
+logger = logging.getLogger("celery.django")
+
+
+### ERROR REPORTING ###
+
+
+@task_failure.connect()
+def celery_task_failure_email(**kwargs):
+    """
+    Celery 4.0+ has no method to send emails on failed tasks.
+    This event handler provides equivalent functionality to Celery < 4.0,
+    and reports truly failed tasks, like those terminated after CELERY_TASK_TIME_LIMIT
+    and those that throw uncaught exceptions.
+    From https://github.com/celery/celery/issues/3389
```

Comment:
Is this an opportunity to start to move away from this reporting method? The reasons for removing the feature in Celery in the linked issue seem like good ones.

Reply:
Ah, did not even think about that once.

Reply:
To me, it's confusing to have some kinds of things that result in error emails and some that don't. If we remove this code, which does sound sensible to do either now or soon, it seems to me that we should simultaneously stop H2O from sending error emails at all.

Reply:
Either way sounds good to me!

Reply:
Happy to work with you all to set up Slack notifications from Sentry next week.

Reply:
Removed. And, since this code included a test of the setup, added a test to the demo task instead.